docs/examples/notebooks/ShapeFactor.ipynb

```diff
@@ -258,8 +247,8 @@
     "    plt.ylim(0,1)\n",
     "\n",
     "def CLsOnePoint(muTest, data, pdf, init_pars, par_bounds):\n",
-    "    clsobs, cls_exp = pyhf.runOnePoint(muTest, data, pdf, init_pars,par_bounds)[-2:]\n",
-    "    return 1./clsobs, [1./x for x in cls_exp]\n",
+    "    cls_obs, cls_exp = pyhf.runOnePoint(muTest, data, pdf, init_pars,par_bounds)[-2:]\n",
+    "    return cls_obs, cls_exp\n",
     "\n",
```

Comment:
I think with this diff we can completely remove this `CLsOnePoint` function and just use `pyhf.runOnePoint`

Reply:
Good point. I'll apply this to everything.

Reply:
This (and all other instances) should now be fixed in commit b4a17d8